Dockerfile

```diff
@@ -168,6 +168,9 @@ RUN pip install --no-cache-dir "numpy-stl" "scikit-image>=0.24.0"
 # Install sparse-dot-mkl
 RUN pip install --no-cache-dir "sparse-dot-mkl"
 
+# Install shapely
+RUN pip install --no-cache-dir "shapely"
```

Comment:
Please add this dependency to changelog

Reply:
It's only in the CI image, so I don't think we need to add